Dockerfile

```diff
@@ -49,3 +49,4 @@ RUN sh /tmp/update_selinon.sh
 RUN cd /tmp/install_deps/ && /tmp/install_deps/apply_patches.sh
 
 RUN yum install -y npm && npm install -g semver-ranger
+RUN pip3 uninstall -y protobuf && pip3 install packaging appdirs && pip3 install --upgrade --no-binary :all: protobuf==3.2.0
```

Comment:
Python requirements are specified in [requirements.txt](https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/requirements.txt), which is generated from [requirements.in](https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/requirements.in) with `pip-compile --output-file requirements.txt requirements.in`.